runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -325,6 +354,15 @@ public void onConfigChanged(ConfigChanged event)
 	@Subscribe
 	public void onMenuOptionClicked(MenuOptionClicked event)
 	{
+		if (config.showAntiPoison()	&& event.getMenuOption().contains("Cure"))
+		{
+			// what was drunk? we don't know.
+			// so, take inv snapshot before your character takes a sip
+			this.prevItems = client.getItemContainer(InventoryID.INVENTORY).getItems().clone();
+			// on inv change: see what has changed. GOTO -> onItemContainerChanged
```

Comment:
This can be found pretty simply by checking where `prevItems` is read; this comment is not needed.
```suggestion
```